vmm/src/device_manager.rs

```diff
@@ -643,7 +654,7 @@ impl DeviceRelocation for AddressManager {
                                 .allocate(
                                     Some(GuestAddress(new_base)),
                                     len as GuestUsize,
-                                    Some(len),
+                                    Some(std::cmp::min(len, 0x10)), // The minimum BAR alignment is 0x10
```

Comment:
Same here use of defined constant PVPANIC_DEVICE_MMIO_ALIGNMENT

Reply:
Ditto.